tests/catch/unit/device/hipGetDeviceAttribute.cc

```diff
@@ -366,3 +369,191 @@ TEST_CASE("Unit_hipDeviceGetAttribute_NegTst") {
+    {hipDeviceAttributeMaxGridDimY, "hipDeviceAttributeMaxGridDimY"},
+    {hipDeviceAttributeMaxGridDimZ, "hipDeviceAttributeMaxGridDimZ"},
+    {hipDeviceAttributeMaxSurface1D, "hipDeviceAttributeMaxSurface1D"},
+    {hipDeviceAttributeMaxSurface2D, "hipDeviceAttributeMaxSurface2D"},
+    {hipDeviceAttributeMaxSurface3D, "hipDeviceAttributeMaxSurface3D"},
+    {hipDeviceAttributeMaxTexture1DWidth, "hipDeviceAttributeMaxTexture1DWidth"},
+    {hipDeviceAttributeMaxTexture1DLinear, "hipDeviceAttributeMaxTexture1DLinear"},
+    {hipDeviceAttributeMaxTexture2DWidth, "hipDeviceAttributeMaxTexture2DWidth"},
+    {hipDeviceAttributeMaxTexture2DHeight, "hipDeviceAttributeMaxTexture2DHeight"},
+    {hipDeviceAttributeMaxTexture3DWidth, "hipDeviceAttributeMaxTexture3DWidth"},
+    {hipDeviceAttributeMaxTexture3DHeight, "hipDeviceAttributeMaxTexture3DHeight"},
+    {hipDeviceAttributeMaxTexture3DDepth, "hipDeviceAttributeMaxTexture3DDepth"},
+    {hipDeviceAttributeMaxThreadsDim, "hipDeviceAttributeMaxThreadsDim"},
+    {hipDeviceAttributeMaxThreadsPerBlock, "hipDeviceAttributeMaxThreadsPerBlock"},
+    {hipDeviceAttributeMaxThreadsPerMultiProcessor,
+     "hipDeviceAttributeMaxThreadsPerMultiProcessor"},
+    {hipDeviceAttributeMaxPitch, "hipDeviceAttributeMaxPitch"},
+    {hipDeviceAttributeMemoryBusWidth, "hipDeviceAttributeMemoryBusWidth"},
+    {hipDeviceAttributeMemoryClockRate, "hipDeviceAttributeMemoryClockRate"},
+    {hipDeviceAttributeComputeCapabilityMinor, "hipDeviceAttributeComputeCapabilityMinor"},
+    {hipDeviceAttributeMultiprocessorCount, "hipDeviceAttributeMultiprocessorCount"},
+    {hipDeviceAttributeName, "hipDeviceAttributeName"},
+    {hipDeviceAttributePageableMemoryAccess, "hipDeviceAttributePageableMemoryAccess"},
+    {hipDeviceAttributePageableMemoryAccessUsesHostPageTables,
+     "hipDeviceAttributePageableMemoryAccessUsesHostPageTables"},
+    {hipDeviceAttributePciBusId, "hipDeviceAttributePciBusId"},
+    {hipDeviceAttributePciDeviceId, "hipDeviceAttributePciDeviceId"},
+    {hipDeviceAttributePciDomainID, "hipDeviceAttributePciDomainID"},
+    {hipDeviceAttributeMaxRegistersPerBlock, "hipDeviceAttributeMaxRegistersPerBlock"},
+    {hipDeviceAttributeMaxRegistersPerMultiprocessor,
+     "hipDeviceAttributeMaxRegistersPerMultiprocessor"},
+    {hipDeviceAttributeMaxSharedMemoryPerBlock, "hipDeviceAttributeMaxSharedMemoryPerBlock"},
+    {hipDeviceAttributeTextureAlignment, "hipDeviceAttributeTextureAlignment"},
+    {hipDeviceAttributeTexturePitchAlignment, "hipDeviceAttributeTexturePitchAlignment"},
+    {hipDeviceAttributeTotalConstantMemory, "hipDeviceAttributeTotalConstantMemory"},
+    {hipDeviceAttributeTotalGlobalMem, "hipDeviceAttributeTotalGlobalMem"},
+    {hipDeviceAttributeWarpSize, "hipDeviceAttributeWarpSize"},
+    {hipDeviceAttributeMemoryPoolsSupported, "hipDeviceAttributeMemoryPoolsSupported"}
+    //  {hipDeviceAttributeVirtualMemoryManagementSupported,
```

Comment:
Can you file a ticket for this?

Reply:
The latest HIP supports this attribute. It is now uncommented.